tests/generation/test_utils.py

```diff
@@ -1610,7 +1610,7 @@ def test_generate_from_inputs_embeds(self, _, num_beams):
                 inputs_dict.pop("pixel_values_images", None)
             #   2.C - No easy fix, let's skip the check that compares the outputs from `input_ids` and `inputs_embeds`
             has_complex_embeds_computation = any(
-                model_name in model_class.__name__.lower() for model_name in ["moshi"]
+                model_name in model_class.__name__.lower() for model_name in ["moshi", "qwen2vl"]
```

Comment:
This flag skips the check that `inputs_embeds = model.get_input_embeddings()(input_ids)` should be equivalent to passing `input_ids` to the model.

This is not true in Qwen2VL, whose computation of `inputs_embeds` is more complex

Reply:
OK for me. Just curious: would this be addressed in the future to make it work

Reply:
Not unless we create a model function to embed inputs, similar to `def get_input_embeddings(self)`.

(I think we should do it eventually, we're starting to have a few models with non-standard inputs embeddings creation!)

Reply:
got it :-) 👍